packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -527,19 +527,25 @@ export default class Meetings extends WebexPlugin {
         LoggerProxy.logger.info('Meetings->createMeeting#Info assuming this destination is a 1:1 or wireless share');
         LoggerProxy.logger.debug(`Meetings->createMeeting#Debug ${err} fetching /meetingInfo for creation.`);
         // We need to save this info for future reference
-        meeting.destination = destination;
+
+        // Collect the meeting info options.
+        const infoOptions = await this.meetingInfo.fetchInfoOptions(destination, type);
```

Comment:
This should be surrounded by a try/catch as well since `fetchInfoOptions` calls `MeetingInfoUtil.generateOptions` which could throw an error.

```js
let infoOptions;
try {
  infoOptions = await
  meeting.destination = infoOptions.destination
}
catch(fetchInfoError) {
  LoggerProxy.logger.info
  meeting.destination = destination
}